doc/release/release_template.rst

```diff
@@ -37,4 +37,4 @@ Deprecations
 
 
 Contributors to this release
-----------------------------
+----------------------------
```

Comment:
Thanks!
I mean there is some difference if GitHub shows it. Probably, the endline character is different or something.